### ISSUE TITLE
Ignoring comments

### ISSUE THREAD

**Issue by jstypka** [maxpumperla/hyperas#7]
So in the model function I often experiment with different configurations commenting in/out different parameters and layers of the model. The thing is, even if you comment out the line with `{{uniform[0,1]}}` it is still gonna be picked up by the parsing script and added into parameter space.

E.g. I get this:
```
    def get_space(): 
        return {
            'GRU': hp.choice('GRU', [256, 512, 1024]),
            'Dropout': hp.uniform('Dropout', 0.0, 0.7),
            'Dense': hp.choice('Dense', [512, 1024]),
            'Dropout_1': hp.uniform('Dropout_1', 0.0, 0.3)
        }
```

For this code:
```
    model = Sequential()
    model.add(GRU(
        {{choice([256, 512, 1024])}},
        input_dim=EMBEDDING_SIZE,
        input_length=SAMPLE_LENGTH,
    ))
    model.add(BatchNormalization())
    model.add(Dropout({{uniform(0, 0.7)}}))

    #model.add(Dense({{choice([512, 1024])}}, activation='relu'))
    
    #model.add(BatchNormalization())
    #model.add(Dropout({{uniform(0, 0.3)}}))

    model.add(Dense(OUTPUT_LAYER))
```

Would it be possible to filter them out in the parser?

**Comment by maxpumperla** [comment]
Hmm, you're right. Thanks for noting this. I'll fix that soon!

**Comment by maxpumperla** [comment]
Hi @jstypka, I fixed this issue in release 0.1.1. Feel free to close the issue when you confirm.

Also, thanks again for your feedback on design, I'm still thinking about how to improve things.
